challenge-13/challenge-13.js

```diff
@@ -1,150 +1,179 @@
-*/
-console.log( '\nAlguns Estados do Brasil:' );
-// ?
-
-/*
-Crie um novo array chamado `nordeste`, que tenha os estados do nordeste.
-*/
-// ?
-
-/*
-Mostre no console os estados do nordeste.
-*/
-console.log( '\nEstados do Nordeste:' );
-// ?
-
-/*
-Remova de `brasil` os estados do `sudeste`, colocando-os em uma variável
-chamada `newSudeste`.
-*/
-// ?
-
-/*
-Adicione os estados do `nordeste` ao array `brasil`. Esses estados devem
-ficar no mesmo nível que os estados já existentes, não em um array separado.
-*/
-// ?
-
-/*
-Mostre no console os estados em `newSudeste`.
-*/
-console.log( '\nEstados em newSudeste:' );
-// ?
-
-/*
-Mostre no console os estados do `brasil`.
-*/
-console.log( '\nAlguns estados do Brasil:' );
-// ?
-
-/*
-usando forEach, percorra o array `brasil` e gere um novo array chamado
-`newBrasil`. Esse array deve ter cada item como um objeto, com as
-propriedades:
-- `id`: que será o índice do array `brasil`,
-- `estado`: que será o estado do array `brasil`.
-*/
-// ?
-
-/*
-Mostre o array `newBrasil` no console
-*/
-console.log( '\nnewBrasil:' );
-// ?
-
-/*
-Percorra o array `brasil` e verifique se os estados tem mais de 7 letras cada,
-atribuindo o resultado à uma variável. Se tiver, mostre no console a frase:
-- "Sim, todos os estados tem mais de 7 letras!"
-Senão, mostre no console:
-- "Nem todos os estados tem mais de 7 letras!"
-*/
-console.log( '\nTodos os estados de `brasil` tem mais de 7 letras?' );
-// ?
-
-/*
-Percorra o array `brasil` e verifique se o Ceará está incluído, atribuindo o
-resultado à uma variável. Se esse estado existir no array, mostrar a frase no
-console:
-- "Ceará está incluído!"
-Senão, mostrar a frase:
-- "Ceará não foi incluído :("
-*/
-console.log( '\nCeará está incluído em `brasil`?' );
-// ?
-
-/*
-Percorra o array `newBrasil` e crie um novo array que some 1 no ID de cada
-objeto desse array, e adicione a frase abaixo na propriedade `estado`:
-- "[ESTADO] pertence ao Brasil."
-Atribua o novo array a uma variável chamada `map`.
-*/
-// ?
-
-/*
-Mostre no console o array criado acima:
-*/
-console.log( '\nnewBrasil agora com mais informações:' );
-// ?
-
-/*
-Filtre o array criado acima, retornando somente os estados que tiverem
-ID par. Atribua o valor à uma variável chamada `filter`.
-*/
-// ?
-
-/*
-Mostre o array filtrado acima no console.
-*/
-console.log( '\nEstados com ID par:' );
-// ?
\ No newline at end of file
+(function(){
+    /*
+    Envolva todo o código desse desafio em uma IIFE.
+    */
+
+    /*
+    Crie um array e mostre no console a representação em String desse array,
+    usando o método visto na aula 13.
+    */
+    console.log( 'O array em formato de string é:' );
+    console.log( [1, 2, 3].toString() );
+
+    /*
+    Crie 2 arrays `sul` e `sudeste`, que serão as regiões do Brasil.
+    Cada array deve conter os estados dessa região.
+    */
+    var sul = [ 'Rio Grande do Sul', 'Paraná', 'Santa Catarina'];
```

Comment:
-1
Procure manter os espaçamentos concisos. Se deixou dentro do array, no início, deixe no final também :)